### ISSUE TITLE
idea: manage borders

### ISSUE THREAD

**Issue by edmundlaugasson** [aardappel/treesheets#46]
There is an idea to be able manage borders in better way like in spreadsheet programs are. It means to be able set (and unset) border (and its style, thickness, color) to any selected cells. Also when setting border thickness to 0 then it would be unset. Currently when wraping parent then there will be a border around the cell but how to remove or edit it - could not find it. Also it seems that setting border thickness does not work for now (64-bit Linux version).

**Comment by aardappel** [comment]
You can edit the cell color and its thickness. You can even set its roundness, though only globally. I doubt editing each side individually is that desirable.

**Comment by edmundlaugasson** [comment]
Yes - each side individually. I found different render styles. But I have to admit that Edit->Set grid border with... does not work. Also - how can I remove current cell border? Or add border to cell or selected cells? Could not find. Therefore I proposed also current idea.
Yes - I found Options->Roundness of grid borders - nice feature.

**Comment by aardappel** [comment]
What exactly "does not work" ? Set grid border width seems to work just fine here.. be sure to have the cell selected that the grid is inside of.

You cannot remove a cell border entirely, at best you can choose a different rendering style (see "layout and render style" menu)

**Comment by edmundlaugasson** [comment]
OK - I see. I meant to remove cell border entirely when it is created e.g. using Insert or F9 (Wrap new parent) key. Or change that border. I discovered that in order to set grid border I have to select parent cell. Selecting the cell with border and changing it does not change the border.
I noticed different rendering styles - good.

**Comment by noyannus** [comment]
>  I discovered that in order to set grid border I have to select parent cell

That is really counter-intuitive. Everywhere in a modern GUI an action acts on the seleted thing, not on some child or other not visibly selected item.

And an option to set each border (N,S,E,W) separately would add greatly to the capability of giving an easy to parse visual orientation. Highly desirable, imo.
